Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -266,12 +272,22 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
 
 void MenuBar::UpdateToolsMenu(bool emulation_started)
 {
-  const bool enable_wii_tools = !emulation_started || !SConfig::GetInstance().bWii;
-  m_perform_online_update_menu->setEnabled(enable_wii_tools);
-  if (enable_wii_tools)
+  m_boot_sysmenu->setEnabled(!emulation_started);
+  m_perform_online_update_menu->setEnabled(!emulation_started);
+
+  if (!emulation_started)
   {
     IOS::HLE::Kernel ios;
     const auto tmd = ios.GetES()->FindInstalledTMD(Titles::SYSTEM_MENU);
+
+    const QString sysmenu_version =
+        tmd.IsValid() ?
+            QString::fromStdString(DiscIO::GetSysMenuVersionString(tmd.GetTitleVersion())) :
+            QStringLiteral("");
+    m_boot_sysmenu->setText(tr("Load Wii System Menu %1").arg(sysmenu_version));
+
+    m_boot_sysmenu->setEnabled(!emulation_started && tmd.IsValid());
```

Comment:
`!emulation_started` can be dropped now that the if condition is `!emulation_started`.